coworking/demo/appointment_type.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="appointment_type_1" model="appointment.type">
+        <field name="is_published" eval="True"/>
+        <field name="resource_ids" eval="[(6, 0, [ref('appointment_resource_1'), ref('appointment_resource_2'), ref('appointment_resource_3'), ref('appointment_resource_4')])]"/>
```

Comment:
I wonder if you need this, because you already do the opposite (filling appointment_type_ids on appointment.resource records in data/
```suggestion
```